src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
I don't think this file is needed to fulfil the brief. Can it be removed?

Reply:
Oh! Sorry, I mean `src/index.js`. Often, when I'm making a comment relating to the file as a whole, I will leave the comment on the first line of the file.

So my previous statement means. I don't think `src/index.js` is needed to fulfil the brief. Can `src/index.js` be removed from this change set? (Please also remove any related files as well e.g `src/index.css`)